app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
Great, I just need to work out how to replicate you NOT seeing it, because I do see it.